### DISCUSSION TITLE
Server map saving...

### DISCUSSION THREAD

**Discussion by VineDiamond** [Pryaxis/TShock#2468] [Q&A] [answered]
I'm using TShock 4.5.5.0 and I want to get rid of the "Server map saving..." announcement in-game. Is there a way to do that?
also the announce save permission in the config is off

**Comment by comdar16** [comment]
> I'm using TShock 4.5.5.0 and I want to get rid of the "Server map saving..." announcement in-game. Is there a way to do that? also the announce save permission in the config is off

try to disable backup map saving message if it is on

**Comment by moisterrific** [comment, marked as answer]
There are two config flags related to this, `AnnounceSave` and `ShowBackupAutosaveMessages`. What you're looking for is probably the second one. 

(from [TShockConfig.cs](https://github.com/Pryaxis/TShock/blob/03c20a98048ee7493d58140c0b8458a3d4399484/TShockAPI/Configuration/TShockConfig.cs#L61))
![image](https://user-images.githubusercontent.com/57187883/136714565-605bad47-9200-4e7c-873d-4447458add80.png)
